pkg/fiddler/fiddler.go

```diff
@@ -126,7 +129,14 @@ func transformField(oldValue interface{}, relation string, local string) (interf
 		case "Source":
 			return oldValue, nil
 		case "ParseTimeDate":
-			return parseDate(oldValue.(string))
+			switch v := oldValue.(type) {
+			case string:
+				return parseDate(oldValue.(string))
```

Comment:
ParseDate currently does Layout based transformation and changes the data type.  Simplify ParseDate to only do the layout based transformation, then change the datatype outside the func.